server/controller-sdk/src/main/java/ai/starwhale/mlops/api/protocol/report/resp/TaskTrigger.java

```diff
@@ -25,9 +25,9 @@
 public class TaskTrigger {
 
     /**
-     * task meta info
+     * unique id for the task
      */
-    private Task task;
+    Long id;
```

Comment:
is jobinfo  necessary send to agent？so agent can upload task's result with common path of jobid

Reply:
every task has it's result path: Task trigger.resultPath